src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -231,6 +231,7 @@ gui.appliedenergistics2.CraftingStatus=Crafting Status
 gui.appliedenergistics2.RemainingOperations=Idle Processors
 gui.appliedenergistics2.FromStorage=Available
 gui.appliedenergistics2.ToCraft=To Craft
+gui.appliedenergistics2.ToCraftRequests=Times Ordered
```

Comment:
```suggestion
gui.appliedenergistics2.ToCraftRequests=Times
```
It looks too long, I think short word would be better here

Reply:
Changed to `Steps`